Tasks/PublishBuildArtifactsV1/publishbuildartifacts.ts

```diff
@@ -71,8 +110,8 @@ async function run() {
             data["containerfolder"] = artifactName;
 
             // add localpath to ##vso command's properties for back compat of old Xplat agent
-            data["localpath"] = pathtoPublish;
-            tl.command("artifact.upload", data, pathtoPublish);
+            data["localpath"] = pathToUpload;
```

Comment:
Please add this update to PR's description

Reply:
Updated